packages/react-components/src/organisms/CalendarList.tsx

```diff
@@ -58,19 +58,27 @@ const orderList = css({
 interface CalendarListProps {
   calendars: ReadonlyArray<CalendarResponse>;
   singleGroup?: boolean;
```

Comment:
Looks like these are exclusive, specifying both doesn't make sense, so the interface would need to change. But instead maybe just pass in the headline and a bool for whether to show the explanation text?